storage/tree/layout.go

```diff
@@ -97,9 +95,11 @@ func (l *Layout) Split(id NodeID) (TileID, *Suffix) {
 	return TileID{Root: NodeID{Path: []byte{}}}, EmptySuffix
 }
 
-// GetTileHeight returns the height of the tile with the passed-in ID.
-func (l *Layout) GetTileHeight(id TileID) int {
-	return l.getStratumAt(id.Root.PrefixLenBits).height
+// TileHeight returns the height of a tile with its root located at the
+// specified depth from the tree root. The result is not defined if rootDepth
+// is not a tile boundary.
```

Comment:
Add TODO to move this back to a state where it takes a structured argument instead of an int.

Reply:
Done.